src/test/java/com/squareup/javapoet/TypeSpecTest.java

```diff
@@ -53,6 +51,7 @@
   private static final String donutsPackage = "com.squareup.donuts";
 
   @Rule public final CompilationRule compilation = new CompilationRule();
+  @Rule public final ExpectedException expectedException = ExpectedException.none();
```

Comment:
Alright, replaced that with a try/catch clause. Would be interesting to hear your opinion on that rule.